docs/HOWTO -- NVIDIA AGX ORIN  on-device kernel build.md

```diff
@@ -0,0 +1,112 @@
+# HOWTO -- NVIDIA AGX ORIN on kernel on-device development process
+
+
+Nvidia is powerful device so developing and something lowlevel as drivers and passthroughs may be fast and efficient when done on board directly.
+
+
+1. Preparational steps -- get a device, plug in monitor and keyboard (in most cases you need DP cable and monitor with DP socket, adapters DP-HDMI will not work before firmware update).
```

Comment:
What does it mean particularly "get"? If it is some kind of requirements (like, what do you need before you start), we can take this sentence out of the list. Or maybe we can add some other requirements here?

Anyway, it should be rewritten like this:

1. Get a device (by the way, what type of the device?), plug-in monitor, and keyboard.

(the next sentence will be better displayed as a note, use https://www.markdownguide.org/basic-syntax/#blockquotes-1)

> In most cases you need a DP cable and monitor with a DP socket, as adapters DP-HDMI do not work before a firmware update.